src/js/components/TeamDetail.js

```diff
@@ -0,0 +1,230 @@
+				<li class="todoapp-container" data-member-id="${_id}">
+					<h2><span><strong>${name}</strong>'s Todo List</span></h2>
+					<div class="todoapp">
+						<section class="input-container">
+							<input class="new-todo" placeholder="할 일을 입력해주세요." autofocus />
+						</section>
+						<section class="main">
+							<ul class="todo-list">
+							${todos}
+							</ul>
+						</section>
+						<div class="count-container">
+							<span class="todo-count">총 <strong>${todoList.length}</strong> 개</span>
+							<ul class="filters">
+								<li><a class="filter selected" href="#all" data-filter="ALL">전체보기</a></li>
+								<li><a class="filter" href="#priority" data-filter="PRIORITY">우선 순위</a></li>
+								<li><a class="filter" href="#active" data-filter="ACTIVE">해야할 일</a></li>
+								<li><a class="filter" href="#completed" data-filter="COMPLETED">완료한 일</a></li>
+							</ul>
+							<button class="clear-completed">모두 삭제</button>
+						</div>
+					</div>
+				</li>`;
+    })
+
+    memberItems += $addUserButtonContainer;
+    $todoappListContainer.innerHTML = memberItems;
+
+    addEvent();
+  };
+
+  const $new = ({_id, contents, isCompleted, priority}) => {
+    return `
+			<li class="todo-list-item ${isCompleted && "completed"}" data-id="${_id}">
+				<div class="view">
+					<input class="toggle" type="checkbox" ${isCompleted && "checked"}/>
+					<label class="label">
+							<div class="chip-container">${setPriority(priority)}</div>
+							${contents}
+					</label>
+					<button class="destroy"></button>
+				</div>
+				<input class="edit" value="${contents}" />
+			</li>`;
+  };
+
+
+  const setPriority = (priority) => {
+    const className = (priority === "FIRST" && "primary") || (priority === "SECOND" && "secondary");
+
+    const item = `
+			<select class="chip select ${className}">
+				<option value="0" ${priority === "NONE" && "selected"}>순위</option>
+				<option value="1" ${priority === "FIRST" && "selected"}>1순위</option>
+				<option value="2" ${priority === "SECOND" && "selected"}>2순위</option>
+			</select>`
+
+    return item;
+  }
+
+
+  const changePriority = async ($chipSelect, value) => {
+    const memberId = $chipSelect.closest(".todoapp-container").dataset.memberId;
+    const itemId = $chipSelect.closest(".todo-list-item").dataset.id;
+    let priority = "";
+
+    if (value === "1") priority = "FIRST";
+    else if (value === "2") priority = "SECOND";
+    else priority = "NONE";
+
+    await Api.putFetch(`/api/teams/${this.teamId}/members/${memberId}/items/${itemId}/priority`, {priority: priority});
+    setMemberList();
+  }
+
+
+  const editItem = ($label) => {
+    const $parentLi = $label.closest(".todo-list-item");
+    const $edit = $parentLi.querySelector(".edit");
+
+    $$(".todo-list li").forEach(li => li.classList.remove("editing"));
+    $parentLi.classList.add("editing");
+
+    $edit.addEventListener("keyup", ({key, currentTarget}) => key === "Enter" && saveEditItem(currentTarget))
```

Comment:
저도 저번에 배웠는데 
`keyup` 이벤트는한글을 입력하고 Enter를 눌렀을 때 Todo Item이 2번 입력이 되는 현상이 있었어요!
그래서 원인을 찾아보다 보니 브라우저에 따라서 현상이 다를 것 이란 글을 확인해서 keyup이벤트를 keypress 이벤트로 대체했을 때 크롬에서 현상이 사라지는 걸 확인했어요.
해당 현상과 관련된 글은 이 [링크 ](https://github.com/vuejs/vue/issues/10277)를 참고했습니다.

```suggestion
    $edit.addEventListener("keypress", ({key, currentTarget}) => key === "Enter" && saveEditItem(currentTarget))
```